source/c/test.c

```diff
@@ -8,7 +8,9 @@
 #include <unistd.h>
 #include <getopt.h>
 
-
+/*
```

Comment:
Add one suggestion for fix.

```suggestion
/*
 * Variables
*/

/*
```